<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightCCSControlGUIForm</class>
 <widget class="GUIFormBase" name="LightCCSControlGUIForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>70</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light CCS Control GUI</string>
  </property>
  <widget class="QDial" name="dialBrightness">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBrightness">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
